pypeit/edgetrace.py

```diff
@@ -3672,9 +3672,11 @@ def nudge_traces(self, trace_cen):
             `numpy.ndarray`_: The nudged traces.
         """
         # Check input
-        if self.par['max_nudge'] is not None and self.par['max_nudge'] <= 0:
-            # Nothing to do
-            return trace_cen
+        if self.par['max_nudge'] is None:
```

Comment:
I'm not following the logic here. I think @kbwestfall should take a look.

Reply:
The default value of `max_nudge` is "None". Previously, a "None" value wasn't caught by this if statement. I think it should be `if max_nudge is None or max_nudge <=0` then do nothing and return. My update is first checking if max_nudge is None, and if not, check if max_nudge is positive.

Reply:
This changes the behavior of the code.  In the original logic, a value of `max_nudge = None` means "nudge the trace as far as you want", whereas now it means "don't nudge the traces."  The latter was achieved in the previous logic by setting `max_nudge = 0` (or a negative number.  The description in `EdgeTracePar` attempts to explain this, but it could be more clear.  Unless there's a failure, I suggest we revert the logic.

Reply:
OK - this makes sense, in which case we need to update the info msg on line 3687 - I'll do this  :-)  Thanks for the clarification @kbwestfall!

Reply:
Good point!  Thanks for correcting the info message.

Reply:
Thanks @kbwestfall. That was my concern that the behavior was being changed.